<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgBox</class>
 <widget class="QDialog" name="DlgBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>361</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>234</red>
        <green>239</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>208</red>
        <green>219</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>234</red>
        <green>239</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>208</red>
        <green>219</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>208</red>
        <green>219</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>208</red>
        <green>219</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Table des box</string>
  </property>
  <property name="windowIcon">
   <iconset resource="synoptux.qrc">
    <normaloff>:/images/Box.png</normaloff>:/images/Box.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTreeWidget" name="treeWidget_Box">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="indentation">
      <number>0</number>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string notr="true">Code</string>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Libelle</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
      <property name="textAlignment">
       <set>AlignHCenter|AlignVCenter|AlignCenter</set>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_code">
       <item>
        <widget class="QLabel" name="label_CodeBqe">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Code box</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_CodeBox">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="maxLength">
          <number>20</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_NomBqe">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Libell√©</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_NomBox">
         <property name="maxLength">
          <number>30</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_coul_b">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Couleur de fond</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_CouleurFond">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_CouleurFondCode"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,0">
       <item>
        <widget class="QLabel" name="label_coult">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Couleur du titre</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_CouleurTitre">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_CouleurTitreCode"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_nbmax">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Nombre de places</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_NbreMaxPat"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Type </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_type">
         <item>
          <property name="text">
           <string>Attente</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Soins</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Absence</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sortie</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_ImageEuros">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>images/Euros.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QPushButton" name="pushButton_ValidBox">
     <property name="text">
      <string>Valider</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="2">
    <widget class="QPushButton" name="pushButton_annuler">
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" rowspan="2">
    <widget class="QPushButton" name="pushButton_QuitterBox">
     <property name="text">
      <string>Quitter</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_CodeBox</tabstop>
  <tabstop>pushButton_ValidBox</tabstop>
  <tabstop>pushButton_annuler</tabstop>
  <tabstop>pushButton_QuitterBox</tabstop>
 </tabstops>
 <resources>
  <include location="synoptux.qrc"/>
 </resources>
 <connections/>
</ui>
